File: python/0009.py
#! /usr/bin/python

from math import pow

prod=0
break_loops=False
for i in range(1, 1000):
    for j in range(1, 1000):
        for k in range(1, 1000):
            if k < j < i and pow(k, 2) + pow(j, 2) == pow(i, 2) and k+j+i == 1000:
                prod = k*j*i
                break_loops = True
                break
        if break_loops:
            break
    if break_loops:
        break

print(prod)

File: php/0009.php
<?php

for($a = 1;$a < 1000; $a++){
	for($b = 1;$b < 1000; $b++){
		for($c = 1;$c < 1000; $c++){
			if($a + $b + $c === 1000 && pow($a,2) + pow($b,2) === pow($c,2)){
                $p = $a*$b*$c;
				echo "$a + $b + $c = $p";
				return;
			}
		}
	}	
}

